eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantSourceEntityRepository.java

```diff
@@ -36,4 +39,14 @@
 
     List<VariantSourceEntity> findByStudyIdOrStudyName(String studyId, String studyName);
 
+    List<VariantSourceEntity> findByStudyIdIn(List<String> studyIds, Pageable pageable);
+
+    long countByStudyIdIn(List<String> studyIds);
+
+    @Query(value = "{fid: {$in : ?0}}}", fields = "{fid : 1, samp : 1}")
```

Comment:
no, as general rule, the apis should be as tight as possible. this allows changes more easily, imagine that a user relies on a api feature that we provide because it's easy but not important. then we change arquitecture, and that field is still not important and difficult, so we break compatibility. If instead we force the user to use the main fields there might be some workaround, but no big deal with internal updates.

Ok, the reason to use the `@Query` was to do a projection. That's actually interesting, from the docs I thought the only way to do that was using another bean with a subset of fields